releases/release-1.30/release-notes/maps/pr-123023-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 123023
+releasenote:
+  text: Fixed kubectl explain to shows enum for field types if they were defined.
```

Comment:
Suggestion: 'The `kubectl explain` command has been updated to show enum values for field types if they were defined.'